traits/util/tests/test_trait_documenter.py

```diff
@@ -97,29 +104,19 @@ def test_get_definition_tokens(self):
 
     def test_add_line(self):
```

Comment:
This test has been rewritten to be a bit less mocky: in particular, it now ends up passing a genuine class to `trait_definition`;  previously it passed `None`.